### PR TITLE
Button success variant

### PR DESCRIPTION
**Pull request by Tom-paqtcom**
A success variant so you can give a positive color to per example a save action if you don't want to use your primary button. Chose to use a background-color because you want to see this button and only a border it's less noticeable, so that's the reason I did not do it like the danger variant.

### CODE REVIEW

**Comment by elwinvaneede** [comment]
@Tom-paqtcom I don't read a clear differencing purpose of when to use this variant. What would you write in https://graphitedesignsystem.com/components/button/#variants? A "success" variant doesn't seem logical to me for a button. "Success" is a result of an action, not an action itself. 

Design System's like Polaris, Spectrum, Carbon, and Duet, don't have a "success" button variant either.

You can always introduce this custom variant in your project, by setting CSS Custom Props for `gr-button[variant="primary"]` or using a class like `gr-button.success`.

File: packages/core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@graphiteds/core",
-  "version": "1.7.0",
+  "version": "1.7.1",
```
Comment (elwinvaneede) [inline packages/core/package.json:3]: No need to do this, `lerna` does this when publishing.

**Comment by Tom-paqtcom** [comment]
> @Tom-paqtcom I don't read a clear differencing purpose of when to use this variant. What would you write in https://graphitedesignsystem.com/components/button/#variants? A "success" variant doesn't seem logical to me for a button. "Success" is a result of an action, not an action itself.
> 
> Design systems like Polaris, Spectrum, Carbon, and Duet, don't have a "success" button variant either.
> 
> You can always introduce this custom variant in your project, by setting CSS Custom Props for `gr-button[variant="primary"]` or using a class like `gr-button.success`.

Bootstrap has it, don't see a reason why a 'danger' variant is much better then.

**Comment by elwinvaneede** [comment]
> Bootstrap has it, don't see a reason why a 'danger' variant is much better then. Maybe 'success' isnt the correct word yes.

Bootstrap is not a Design System.

And "Danger" has a very clear purpose: "Use when the action will delete data or be otherwise difficult to recover from."

**Comment by Tom-paqtcom** [comment]
> > Bootstrap has it, don't see a reason why a 'danger' variant is much better then. Maybe 'success' isnt the correct word yes.
> 
> Bootstrap is not a design system.
> 
> And "Danger" has a very clear purpose: "Use when the action will delete data or be otherwise difficult to recover from."

Not agreeing but fine

**Pull request closed without merging**